docs/durable_execution/temporal.md

```diff
@@ -160,6 +160,54 @@ When `TemporalAgent` dynamically creates activities for the wrapped agent's mode
 
 Other than that, any agent and toolset will just work!
 
+### Model Selection at Runtime
+
+By default, a `TemporalAgent` uses the model that was set on the wrapped agent. However, you can register additional models and select between them at runtime inside workflows using the `additional_models` parameter:
+
+```python {title="multi_model_temporal.py" test="skip"}
+from temporalio import workflow
+
+from pydantic_ai import Agent
+from pydantic_ai.durable_exec.temporal import TemporalAgent
+from pydantic_ai.models.openai import OpenAIModel
+
+# Create models
+default_model = OpenAIModel('gpt-4o')
+fast_model = OpenAIModel('gpt-4o-mini')
+reasoning_model = OpenAIModel('o1')
```

Comment:
Let's use more recent models from different providers to illustrate that they don't need to be from the same provider